src/connector/src/connector_common/iceberg/mod.rs

```diff
@@ -79,15 +79,31 @@ pub struct IcebergCommon {
     /// A Bearer token which will be used for interaction with the server.
     #[serde(rename = "catalog.token")]
     pub token: Option<String>,
-    /// `oauth2-server-uri` for accessing iceberg catalog, only applicable in rest catalog.
+    /// `oauth2.server.uri` for accessing iceberg catalog, only applicable in rest catalog.
     /// Token endpoint URI to fetch token from if the Rest Catalog is not the authorization server.
-    #[serde(rename = "catalog.oauth2-server-uri")]
+    #[serde(rename = "catalog.oauth2.server.uri")]
     pub oauth2_server_uri: Option<String>,
     /// scope for accessing iceberg catalog, only applicable in rest catalog.
     /// Additional scope for OAuth2.
     #[serde(rename = "catalog.scope")]
     pub scope: Option<String>,
 
+    /// The signing region to use when signing requests to the REST catalog.
+    #[serde(rename = "catalog.rest.signing.region")]
+    pub rest_signing_region: Option<String>,
+
+    /// The signing name to use when signing requests to the REST catalog.
+    #[serde(rename = "catalog.rest.signing.name")]
+    pub rest_signing_name: Option<String>,
+
+    /// Whether to use SigV4 for signing requests to the REST catalog.
+    #[serde(
+        rename = "catalog.rest.sigv4.enabled",
+        default,
+        deserialize_with = "deserialize_optional_bool_from_string"
+    )]
+    pub rest_sigv4_enabled: Option<bool>,
```

Comment:
The java config is `rest.signing-region`, `rest.sigv4-enabled`, `rest.secret-access-key`. I think we are abusing `.` in our config (a long existing problem...

Reply:
Indeed, at the beginning I wanted to use `est.signing-region` instead of `est.signing.region` however, the parser couldn't handle `-` when parsing identifiers. 😢

Reply:
🤯

Reply:
double quote is ok: 🤔
```sql
dev=> create table t(x int) with ("a-1"='b');
CREATE_TABLE
```
Alternatively, what about using `_`?

Reply:
Using `_` sounds good, because we have `primary_key` which use `_` as well.